sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/README.md

```diff
@@ -59,7 +59,10 @@ To quickly create the needed resources in Azure and to receive connection string
 
 - [Process events in batches](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample09_ProcessEventsByBatch.cs)  
   An example of grouping events into batches for downstream processing.
-  
+
+- [Run in Azure Stack](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample10_RunningInAzureStack.cs)
```

Comment:
Should we include something about "using an earlier version of Azure Storage" or something similar?   At some point, Azure Stack is likely to upgrade the storage, after all.   I'm not sure about this, honestly, just raising for thoughts.

Reply:
I was expecting to completely remove the sample in this scenario. Do you think it could be useful if we kept it?

Reply:
Potentially.  The government clouds are sometimes a version or so behind the public cloud for services and don't necessarily evolve at the same rate, so this technique could be used for mismatches there as well.